ui/components/multichain/pages/page/page.js

```diff
@@ -17,16 +19,27 @@ export const Page = ({
 }) => {
   return (
     <Box
-      display={Display.Flex}
       width={BlockSize.Full}
-      flexDirection={FlexDirection.Column}
-      style={{ height: '100%' }}
-      className={classnames('multichain-page', className)}
-      {...props}
+      height={BlockSize.Full}
```

Comment:
Looks wonderful! Just one small question. Why do we need this height and width here if we are adding px values

Reply:
These are default settings. I believe the px values you're talking about are specifically the send page overwriting the defaults. I think it's safe to assume 100% here